python/test/test_api_models/test/config.pbtxt

```diff
@@ -0,0 +1,28 @@
+name: "test"
+backend: "python"
+input [
+{
+  name: "text_input"
+  data_type: TYPE_STRING
+  dims: [ -1 ]
+  optional: true
+},
+{
+  name: "fp16_input",
+  data_type: TYPE_FP16
+  dims: [-1,1],
+  optional: true
+}
+]
+output [
+{
+  name: "text_output"
+  data_type: TYPE_STRING
+  dims: [ -1 ]
+},
+{
+  name: "fp16_output",
+  data_type: TYPE_FP16
+  dims: [-1,1]
+}
+]
```

Comment:
do you want these I/O defined here too? or rely on the autocomplete in model.py as single source of truth?